components/scream/scripts/test_all_scream.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, cxx, kokkos=None, submit=False, parallel=False, fast_fail=Fal
         expect(not (self._baseline_ref and self._baseline_dir),
                "Makes no sense to specify a baseline generation commit if using pre-existing baselines ")
 
+        self._custom_cmake_opts.append("EKAT_DEFAULT_BFB=ON")
```

Comment:
Is this intended?

Reply:
This will be resolved in https://github.com/E3SM-Project/scream/pull/629. Just here so I could see if the tests pass.